charts/rancher-csp-adapter/0.1.0+up0.1.0-rc5/templates/configmap.yaml

```diff
@@ -0,0 +1,6 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ .Chart.Name }}-status
+  namespace: cattle-csp-adapter-system
+data: {}
```

Comment:
nit: newline

Also, do you need to include the data field if it’s empty anyways?

Reply:
Tested, and turns out no.